pymongo/pool.py

```diff
@@ -1377,9 +1373,9 @@ def __init__(self, address: _Address, options: PoolOptions, handshake: bool = Tr
         # LIFO pool. Sockets are ordered on idle time. Sockets claimed
         # and returned to pool from the left side. Stale sockets removed
         # from the right side.
-        self.conns: collections.deque = collections.deque()
+        self.available_conns: collections.deque = collections.deque()
```

Comment:
It's confusing to have a variable named `conns` represent the connections available for checkout from the pool if we also have a variable called `active_conns` that represents a different set of connections. Changing `conns` to `available_conns` makes the distinction and function of each more clear.

Reply:
I disagree because the same confusion exists between "available_conns" and "active_conns" but regardless now that we have active_conns_context can we rename this back to "conns"?

Reply:
Sure, fair enough.